packages/nextjs/package.json

```diff
@@ -31,6 +31,7 @@
     "next": "14.1.3",
     "next-themes": "^0.2.1",
     "nprogress": "^0.2.0",
+    "prettier-standalone": "^1.3.1-0",
```

Comment:
for the others please note new `yarn install` needed @Nadai2010 @iossocket @0xquantum3labs @jrcarlos2000